lib/signalfx/tracing/sfx_logger.rb

```diff
@@ -0,0 +1,38 @@
+require 'logger'
+
+module SignalFx
+  module Tracing
+    class SFxLogger < Logger
+
+      attr_accessor :log
+
+      LOG_LEVELS = {
+        'unknown' =>Logger::UNKNOWN,
+        'fatal' => Logger::FATAL,
+        'error' =>  Logger::ERROR,
+        'warn' => Logger::WARN,
+        'info' => Logger::INFO,
+        'debug' => Logger::DEBUG
+      }.freeze
+
+      def initialize (dirname = ENV['SIGNALFX_LOG_DIRECTORY'] || 'sfx_logs',
```

Comment:
`/var/log/signalfx`  is a more standardized default directory.  We should also provide documentation for these new options.

Reply:
Still looking into the permission issues inherent in creating a directory in `/var`(if need be)